packages/salesforcedx-vscode-core/src/commands/forceCreateManifest.ts

```diff
@@ -5,16 +5,47 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
+import { LibraryCommandletExecutor } from '@salesforce/salesforcedx-utils-vscode/out/src';
+import { ContinueResponse } from '@salesforce/salesforcedx-utils-vscode/out/src/types';
 import { ComponentSet } from '@salesforce/source-deploy-retrieve';
 import * as fs from 'fs';
 import { join, parse } from 'path';
 import { format } from 'util';
 import * as vscode from 'vscode';
+import { OUTPUT_CHANNEL } from '../channels';
 import { nls } from '../messages';
 import { getRootWorkspacePath } from '../util';
+import { FilePathGatherer, SfdxCommandlet, SfdxWorkspaceChecker } from './util';
 
 const DEFAULT_MANIFEST = 'package.xml';
 
+export class ManifestCreateExecutor extends LibraryCommandletExecutor<string> {
+  private sourcePaths: string[];
+  private responseText: string | undefined;
+  constructor(sourcePaths: string[], responseText: string | undefined) {
+    super(
+      nls.localize('force_create_manifest'),
+      'force_create_manifest',
```

Comment:
can we pull this out into a constant?

Reply:
This is a common theme across all executors. This is part of the whole story to consolidate telemetry constants to a separate file. Until then, I can definitely extract into a constant in the local file.